nilearn/signal.py

```diff
@@ -252,10 +258,10 @@ def butterworth(signals, sampling_rate, low_pass=None, high_pass=None,
     else:
         critical_freq = critical_freq[0]
 
-    b, a = signal.butter(order, critical_freq, btype=btype)
+    b, a = sp_signal.butter(order, critical_freq, btype=btype, output='ba')
```

Comment:
Importing as sp_signal may not require I suppose ?

Reply:
helps clarifying its different from input parameter signals. I was gonna call it scipy_signal, but then it seemed too long :)